Makefile

```diff
@@ -38,9 +38,6 @@ format-fix:
 	rm -rf src/amalgamation/*
 	python3 scripts/format.py --all --fix --noconfirm
 
-format:
-	python3 scripts/format.py --all --fix --noconfirm
```

Comment:
I added this because somehow our `main` didn't have correct formatting
I use `make format-main` to avoid reformatting everything normally - but `make format` doesn't work correctly for the case where this *is* necessary

So I would prefer to keep this, the `extension-ci-tools` format is not doing what it's supposed to, I prefer overriding it with this.

Reply:
does `format-fix` not do the exact same thing?
And you also can use `make format-head` to just fix files that have changed that are tracked right?